brave/brave-core
# Adds uphold card

Resolves https://github.com/brave/brave-browser/issues/10838

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- connect KYC uphold account
- you should see uphold card on rewards page

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
